t/integration/test_loader.py

```diff
@@ -0,0 +1,22 @@
+from celery import shared_task
+
+
+@shared_task()
+def dummy_task(x, y):
+    return x + y
+
+
+class test_loader:
+    def test_autodiscovery(self, manager):
+        # Arrange
+        expected_package_name, _, module_name = __name__.rpartition('.')
+        unexpected_package_name = 'nonexistent.package.name'
```

Comment:
This actually caught a bug I fixed in 3e405b1.

Reply:
thanks!